build/format-changelog.mjs

```diff
@@ -5,14 +5,16 @@
 import { readFile, writeFile } from 'node:fs/promises'
 import { join } from 'node:path'
 
+/* eslint-disable no-console */
+
 console.info('🔎 checking format of CHANGELOG.md')
 
 const file = join(import.meta.dirname, '..', 'CHANGELOG.md')
 const content = await readFile(file, { encoding: 'utf-8' })
 
 const formatted = content.replaceAll(
 	/by @([^ ]+) in ((https:\/\/github.com\/)nextcloud-libraries\/nextcloud-vue\/pull\/(\d+))/g,
-	'[\#$4]($2) \([$1]($3$1)\)'
+	'[#$4]($2) ([$1]($3$1))',
```

Comment:
Diverged here from main as it should include the escaping backslash:

```suggestion
	'[\\#$4]($2) \\([$1]($3$1)\\)',
```

Reply:
I don't understand, why we need a slash here.
But I'll add it to have the same as on the `main` branch.